### PR TITLE
Feat implement get internet speed dart

### PR DESCRIPTION
**Pull request by GunjanDhanuka**
:heavy_check_mark: Dart implementation of the getInternetSpeed function as mentioned in #89 . This function uses Speedtest CLI to check the download and upload speed of the server. The results are printed as the output of the function in the units of Mbps.

### CODE REVIEW

File: dart/getInternetSpeed/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# 📧 getInternetSpeed
+Dart implementation to check the download and upload speed of the server. This code checks the speed using the [SpeedTest CLI](https://www.speedtest.net/apps/cli).
+
+## 📝 Environment Variables
+No environment variables required for this one
+
+
```
Comment (lohanidamodar) [inline dart/getInternetSpeed/README.md:7]: Remove extra line
```suggestion
```

File: dart/getInternetSpeed/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## 🚀 Building and Packaging
+
+To package this as a cloud function, follow these steps.
+
+**Install the SpeedTest CLI according to your distribution from [this link](https://www.speedtest.net/apps/cli)!**
+
+```bash
+$ cd demos-for-functions/dart/getInternetSpeed
+$ dart pub get
```
Comment (lohanidamodar) [inline dart/getInternetSpeed/README.md:16]: need an step to update PUB_CACHE environment to make sure pub dependencies are kept in `.appwrite` folder and folder structure should contain `.appwrite/` folder. Check existing dart function demo.

File: dart/getInternetSpeed/main.dart [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import 'dart:convert';
+import 'dart:io';
+
+// void main() {
```
Comment (lohanidamodar) [inline dart/getInternetSpeed/main.dart:4]: Please remove if this is not necessary. Thank you.

File: dart/getInternetSpeed/pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+name: getInternetSpeed
+version: 1.0.0
+description: "A appwrite function to get the internet speed of the server."
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  dart_appwrite: ^0.6.1
```
Comment (lohanidamodar) [inline dart/getInternetSpeed/pubspec.yaml:9]: Is this package used anywhere?

**Review by lohanidamodar — CHANGES_REQUESTED** [review]

File: dart/getInternetSpeed/pubspec.yaml (hunk 4 above)
Comment (GunjanDhanuka) [reply dart/getInternetSpeed/pubspec.yaml]: No it isn't. I have updated it! Thanks!

File: dart/getInternetSpeed/main.dart (hunk 3 above)
Comment (GunjanDhanuka) [reply dart/getInternetSpeed/main.dart]: Done!

File: dart/getInternetSpeed/README.md (hunk 2 above)
Comment (GunjanDhanuka) [reply dart/getInternetSpeed/README.md:16]: Done! Thanks a lot! I am new to contributing too appwrite hence these details got missed :sweat_smile:

**Comment by GunjanDhanuka** [comment]
> heavy_check_mark Dart implementation of the getInternetSpeed function as mentioned in #89 . This function uses Speedtest CLI to check the download and upload speed of the server. The results are printed as the output of the function in the units of Mbps.

I have made the corrections as suggested! Please review it

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
@GunjanDhanuka please there is an unintended removal of `python/.gitkeep`, please fix it. Thank you

**Comment by Meldiron** [comment]
Also, please rename the folder from `getInternetSpeed` to `get-internet-speed` to stay  consistent with the rest of the repo.

**Comment by GunjanDhanuka** [comment]
> @GunjanDhanuka please there is an unintended removal of `python/.gitkeep`, please fix it. Thank you

That file was an empty file created since I mistakenly started working on Python implementation on the same branch. That is why I deleted that file. Do I still need to add that empty file?

**Comment by GunjanDhanuka** [comment]
> Also, please rename the folder from `getInternetSpeed` to `get-internet-speed` to stay consistent with the rest of the repo.

Done! :+1:

**Comment by GunjanDhanuka** [comment]
> > @GunjanDhanuka please there is an unintended removal of `python/.gitkeep`, please fix it. Thank you
> 
> That file was an empty file created since I mistakenly started working on Python implementation on the same branch. That is why I deleted that file. Do I still need to add that empty file?

Nevermind, I added the ```python/.gitkeep``` file again since it was in the ```master``` branch

File: dart/get-internet-speed/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# 📧 getInternetSpeed
```
Comment (lohanidamodar) [inline dart/get-internet-speed/README.md:1]:
```suggestion
# 📧 Get Internet Speed
```

File: dart/get-internet-speed/pubspec.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+name: getInternetSpeed
+version: 1.0.0
+description: "A appwrite function to get the internet speed of the server."
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
```
Comment (lohanidamodar) [inline dart/get-internet-speed/pubspec.yaml:8]:
```suggestion
```
As there are not dependencies we can remove this. Make sure it works after removing this.

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
@GunjanDhanuka great work. Some changes.
Also there are two copies of every file in two different folders. Make sure to remove duplicates!

File: dart/get-internet-speed/README.md (hunk 5 above)
Comment (GunjanDhanuka) [reply dart/get-internet-speed/README.md]: Done!

File: dart/get-internet-speed/pubspec.yaml (hunk 6 above)
Comment (GunjanDhanuka) [reply dart/get-internet-speed/pubspec.yaml]: Done!

**Comment by GunjanDhanuka** [comment]
> @GunjanDhanuka great work. Some changes. Also there are two copies of every file in two different folders. Make sure to remove duplicates!

@lohanidamodar I have made the requested changes! Can you please review them?

**Comment by lohanidamodar** [comment]
@GunjanDhanuka please also add screenshot of successful execution and output.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Please include a screenshot of successful execution and its output from Appwrite Console. Without that, we cannot proceed to merge the PR.

**Comment by christyjacob4** [comment]
@GunjanDhanuka Closing this PR since there hasn't been any activity in a while. Feel free to reopen it when you're ready to start working on it again.

**Pull request closed without merging**